bin/citgm-all.js

```diff
@@ -97,6 +119,10 @@ function runCitgm (mod, name, next) {
   const runner = citgm.Tester(name, options);
   let bailed = false;
 
+  if (checkTags(app, mod, name)) {
+    return next(); // Skip this module
```

Comment:
How else would you do it? If you do 

```js
return;
next();
```

then next will never get called right? Are you suggesting:

```js
next();
return;
```